### DISCUSSION TITLE
Feature Request - Built in horizontal scroll bar options near text input

### DISCUSSION THREAD

**Discussion by xinle1030** [flyerhq/flutter_chat_ui#188] [Ideas & Requests] [answered]
![69318B11-6B39-4716-A039-24945D2EE4BC](https://user-images.githubusercontent.com/61077925/148313050-a7fa75b7-4f3a-409e-9271-9e34774a878a.jpeg)

It would be nice if we could have a horizontal scroll bar to place default suggestion chips near the text input to the user as shown in the picture before.

**Comment by demchenkoalex** [comment]
To be added :)

**Reply by kristellewidjaja** [reply]
@demchenkoalex I noticed that this request was posted last year. I would also like to have this feature for my mobile app. I want to be able to tap on the suggestion input chips and have default text entered in the message text field for the user. 

Any updates on this feature? I was thinking of using [input chips](https://api.flutter.dev/flutter/material/InputChip-class.html)

Thanks!

**Reply by demchenkoalex** [reply]
There is `listBottomWidget` I think this might be it, where you could put your chips

**Reply by demchenkoalex** [reply]
I don't know why message disappeared, but I read it in email, you have access to the `InputOptions` where you can find a controller, I guess you can use it to control the text. https://github.com/flyerhq/flutter_chat_ui/blob/main/lib/src/widgets/input/input.dart#L248

**Reply by demchenkoalex** [reply, marked as answer]
released in [v2.0.0-dev.2](https://github.com/flyerhq/flutter_chat_ui/releases/tag/2.0.0-dev.2)
